docs/ref/contrib/gis/gdal.txt

```diff
@@ -1117,16 +1117,39 @@ blue.
         >>> rst = GDALRaster('/path/to/your/raster.tif', write=False)
         >>> rst.name
         '/path/to/your/raster.tif'
-        >>> rst.width, rst.height            # This file has 163 x 174 pixels
+        >>> rst.width, rst.height  # This file has 163 x 174 pixels
         (163, 174)
-        >>> rst = GDALRaster({'srid': 4326, 'width': 1, 'height': 2, 'datatype': 1
-        ...                   'bands': [{'data': [0, 1]}]}) # Creates in-memory raster
+        >>> rst = GDALRaster({  # Creates an in-memory raster
+        ...     'srid': 4326,
+        ...     'width': 4,
+        ...     'height': 4,
+        ...     'datatype': 1,
+        ...     'bands': [{
+        ...         'data': (2, 3),
+        ...         'offset': (1, 1),
+        ...         'size': (2, 2),
+        ...         'shape': (2, 1),
+        ...         'nodata_value': 5,
+        ...     }]
+        ... })
         >>> rst.srs.srid
         4326
         >>> rst.width, rst.height
-        (1, 2)
+        (4, 4)
         >>> rst.bands[0].data()
-        array([[0, 1]], dtype=int8)
+        array([[5, 5, 5, 5],
+               [5, 2, 3, 5],
+               [5, 2, 3, 5],
+               [5, 5, 5, 5]], dtype=uint8)
+
+    .. versionchanged:: 1.11
```

Comment:
The entire note can be removed with the versionchanged annotation. I think the complexity of the `ds_input` argument has grown and would benefit from its own section in the docs. I can start work on this. Do you think adding another subsection to the  [Raster Data Objects](https://docs.djangoproject.com/en/1.10/ref/contrib/gis/gdal/#raster-data-objects) would be appropriate?